operations/patch_util.go

```diff
@@ -176,6 +179,8 @@ func (p *patchParams) validatePatchCommand(ctx context.Context, conf *ClientSett
 				fmt.Printf("warning - failed to set default variants: %v\n", err)
 			}
 		}
+	} else if len(p.Variants) != 0 && p.Alias != "" {
+		fmt.Printf("warning - both alias and variants passed in, defaulting to using %v alias\n", p.Alias)
```

Comment:
I think it's desirable to allow a user to specify an alias and expand that to include more variants/tasks

Reply:
while we're here we should probably change these Printf's to use the logger.